docs/insomnia/introduction-to-plugins.md

```diff
@@ -69,15 +69,15 @@ Example: Plugin package.json file
     },
 
     // Force plugin hub and other entities to show specific author details
-    // Useful for teams / organizations who work on same plugin
+    // Useful for teams and organizations who work on the same plugin
     "publisher": {
       "name": "YOUR NAME HERE", // Plugin publisher name, displayed on plugin hub
-      "icon": "https://...", // Plugin publisher avatar /  icon, absolute url
+      "icon": "https://...",    // Plugin publisher avatar or icon, absolute url
     },
 
     // Plugin Application Support
     // semver target supported, see https://semver.npmjs.com/ for more details
-    // * targets all versions
+    // * designates that all versions are targeted
```

Comment:
I need to double check if this is even valid anymore. We no longer have the designer/core separation (we only have core), and I don't think this field is actually used by any logic.

Reply:
We should be okay to remove this. I checked in both the app and on the marketing website, and it appears this data is never used. More details in this cleanup PR on the website: https://github.com/Kong/insomnia-website/pull/11.

I've also raised a cleanup PR in the app repo: https://github.com/Kong/insomnia/pull/4023.